src/test/java/org/gaul/httpbin/HttpBinTest.java

```diff
@@ -43,13 +43,13 @@ public final class HttpBinTest {
 
     @Before
     public void setUp() throws Exception {
-        httpBin = new HttpBin(httpBinEndpoint);
+        httpBin = new HttpBin("127.0.0.1", 8001, 0, "");
```

Comment:
What is the intent here?  Previously the tests used a zero port which selects a free port so it does not conflict with other running services.  Please revert.